Procurement/ViewModel/ItemHoverViewModel.cs

```diff
@@ -104,6 +118,18 @@ public ItemHoverViewModel(Item item)
                 ExperienceDenominator = gem.ExperienceDenominator;
             }
 
+            if (IsIncubatorProgressVisible)
+            {
+                IncubatorNumerator = item.IncubatedDetails.Progress;
+                IncubatorDenominator = item.IncubatedDetails.Total;
+                Incubating = $"Incubating {item.IncubatedDetails.Name}";
+                IncubationLevel =  $"Level {item.IncubatedDetails.Level}+ Monster Kills";
+                if (Item.IncubatedDetails.Total > 0 )
+                {
+                    IncubatorProgress = double.Parse(item.IncubatedDetails.Progress.ToString()) / double.Parse(item.IncubatedDetails.Total.ToString());
```

Comment:
We don't need to convert the values to strings first, do we?  We could simply convert one from an `int` to a `double` and get a `double` result.

Reply:
Just learn about "Convert.ToDouble(anInt)" thanks to this comment.

Thanks